### DISCUSSION TITLE
请问一下能否挂载本地文件夹

### DISCUSSION THREAD

**Discussion by 0xdra** [spencerwooo/onedrive-vercel-index#162] [Q&A] [answered]
暂时只会本地运行项目，挂载OneDrive的文件，想请教下怎样挂载本地的文件夹，想放到服务器上挂载服务器的文件

**Comment by spencerwooo** [comment, marked as answer]
不能，项目目前和 OneDrive API 深度耦合。

**Reply by 0xdra** [reply]
> 不能，项目目前和 OneDrive API 深度耦合。

好的，多谢解答～

**Comment by myl7** [comment]
PR #185
支持了 nginx autoindex 来 serve 本地文件，暂还不支持 mount 到某一目录但之后可以加。
目前这个实现还很粗糙，希望能获得 feedback 来看看要改进的话怎么改进。
